src/main/scala/tech/cryptonomic/conseil/Lorre.scala

```diff
@@ -45,7 +45,7 @@ object Lorre extends App with LazyLogging {
             case Success(_) => logger.info(s"Wrote ${blocks.size} blocks to the database.")
             case Failure(e) => logger.error(s"Could not write blocks to the database because $e")
           }
-          Await.result(dbFut, Duration.Inf)
+          Await.result(dbFut, Duration.apply(conf.getInt("dbAwaitTimeInSeconds"), SECONDS))
```

Comment:
Can we evaluate `conf.getInt("dbAwaitTimeInSeconds")` once up top so it fails fast if there us anything incorrect, then use a variable in all these function calls?